modules/guides/pages/bloblang/methods.adoc

```diff
@@ -1867,6 +1866,21 @@ root.id = this.id.string()
 # Out: {"id":"228930314431312345"}
 ```
 
+=== `timestamp`
+
+Attempts to parse a value into a timestamp. You can also provide an optional argument. The argument is returned when the value cannot be parsed into a timestamp.
```

Comment:
```suggestion
Attempts to parse a value into a `timestamp`. You can also provide an optional argument. The argument is returned when the value cannot be parsed into a `timestamp`.
```

Reply:
It's a good idea to call out what's the timestamp format too

Reply:
@mihaitodor. Can you give details of the timestamp format?

Reply:
In order to be parsed as a timestamp a value needs to be one of the following:
- An integer representing a unix timestamp in seconds
- A floating point number representing a unix timestamp in seconds
- A string in RFC3339 format

Reply:
Updated.